src/Symfony/Component/DependencyInjection/composer.json

```diff
@@ -29,6 +29,7 @@
     "suggest": {
         "symfony/yaml": "",
         "symfony/config": "",
+        "symfony/expression-language": "",
```

Comment:
well, look at the existing one for ``symfony/proxy-manager-bridge`` rather than for ``symfony/yaml`` :smile: